lib/inferno/dsl/fhir_resource_validation.rb

```diff
@@ -193,8 +193,14 @@ def resource_is_valid?(resource, profile_url, runnable, add_messages_to_runnable
                 'Error occurred in the validator. Review Messages tab or validator service logs for more information.'
         end
 
+        # @private
+        def exclude_unresolved_url_message
+          proc { |message| message.message.match?(/could not be resolved/i) }
```

Comment:
Apologies if this wasn't clear in the ticket, but the message needs to be more specific to just "URL value * does not resolve", for example as seen in the US core test kit:

https://github.com/inferno-framework/us-core-test-kit/blob/3f9dcb6ec7d7e08413992e5f6d757a749bcca083/lib/us_core_test_kit/generated/v3.1.1/us_core_test_suite.rb#L67

Reply:
updated